packages/extension-browser/src/devtools/views/controls/summary.css

```diff
@@ -0,0 +1,13 @@
+.summary {
```

Comment:
Probably should rename the class `.root` for consistency with our other controls.